src/jit/compiler.cpp

```diff
@@ -7225,8 +7249,8 @@ void JitTimer::PrintCsvHeader()
 
         fprintf(fp, "\"Total Cycles\",");
         fprintf(fp, "\"CPS\"\n");
-        fclose(fp);
     }
+    fclose(fp);
```

Comment:
Looks like you're closing the read handle, but not the write handle. I assume _waccess() didn't work for some reason?